install/install.qmd

```diff
@@ -105,90 +106,136 @@ For more information, see the
 :::::
 
 ::::
 :::
 
 <script src="installer.js"></script>
 
-## In-Depth Instructions
 
-### Platform
 
-#### Linux
+::: {.callout-note}
+Conda simplifies the installation process by ensuring that all required libraries and tools are compatible with each other
+and is available for Linux, Mac, and Windows platforms.
 
-- which C++
+You can either install [miniconda](https://docs.conda.io/en/latest/miniconda.html), a free, minimal installer for conda
+or you can get the full [Anaconda](https://docs.continuum.io/anaconda/) system
+which provides graphical installer wizards for [MacOS](https://www.anaconda.com/products/distribution#macos){target="_blank"}
+and [Windows](https://www.anaconda.com/products/distribution#windows){target="_blank"} users.
 
-#### MacOS
+Julia users can install [Conda.jl](https://github.com/JuliaPy/Conda.jl){target="_blank"}.
+:::
 
-- Xcode
+## Prerequisite:  C++17
 
-#### Windows
 
-- Conda
-- RTools
+Stan requires a C++17 compiler and the GNU Make utility.
 
-## Other packages and environments {#other-packages}
+* On Linux, these are bundled into the meta-package `build-essential`.
+To install, run command:\
+`sudo apt-get install build-essential`
 
-#### Google Colab
 
-#### Mathematica, Stata, etc?
+* On Mac, these are included with Xcode.
+Install Xcode from the App Store and then run command:\
+`xcode-select --install`
 
-## Troubleshooting
+* On Windows 10, we strongly recommend first installing Anaconda or miniconda and then using the conda installer to be CmdStanR or RStan,
+since the conda packages include a C++ toolchain.\
+Alternatively, R Users can get [Rtools](https://cran.r-project.org/bin/windows/Rtools/){target="_blank"} from CRAN,
+or try to install it from within CmdStanR using the `install_toolchain` method.
 
-### C++ Versions
 
-### C++ Libraries
+## Local CmdStan installations for CmdStanPy, CmdStanR, and Stan.jl
 
-- Boost
-- Sundials
-- TBB
+CmdStanPy, CmdStanR and require a local CmdStan installation and provide scripts and methods to manage this.
+The default installation location is in the user's home directory and is named `.cmdstan` (a hidden directory).
+This directory contains one or more versions of CmdStan.
+Both interfaces provide the following functions to manage the CmdStan installations:
+
+* `install_cmdstan` - downloads, installs, and builds the latest CmdStan relese or specified release
+* `rebuild_cmdstan` - rebuild the specified release.  On Mac, often required after an Xcode update.
+* `show_versions` - shows all installed CmdStan versions
+* `cmdstan_path` - shows which version of CmdStan is being used
+* `set_cmdstan_path` - specify which version of CmdStan to use.
 
-### GPU Support
+For Stan.jl, use Conda.jl or conda to install CmdStan.
 
+## Other Stan Interfaces {#other-packages}
 
+* Google Colab - prebuilt CmdStan binaries for Google Colab are available from the [GitHub CmdStan releases page](https://github.com/stan-dev/cmdstan/releases){target="_blank"}.
```

Comment:
We probably need to explain more about how to use these - and that you don't need to, just that it's faster

Reply:
added a note - but not full instructions.